src/Resources/config/services.xml

```diff
@@ -63,9 +66,11 @@
         </service>
         <service id="Adyen\Shopware\ScheduledTask\Webhook\WebhookHandlerFactory">
             <argument type="service" id="Adyen\Shopware\Service\CaptureService"/>
+            <argument type="service" id="Adyen\Shopware\Service\AdyenPaymentService"/>
             <argument type="service" id="Adyen\Shopware\Service\RefundService"/>
             <argument type="service"
                       id="Shopware\Core\Checkout\Order\Aggregate\OrderTransaction\OrderTransactionStateHandler"/>
+            <argument type="service" id="Adyen\Shopware\Service\Repository\AdyenPaymentRepository" />
```

Comment:
Calling the repository `AdyenPaymentRepository` here wouldn't be a good idea. `AdyenPaymentService` has already been injected to this class. What do you think about creating the related method in the service file and removing the repository dependency from here?